packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -609,6 +609,31 @@ export function createLodestarMetrics(
         help: "Time elapsed between block slot time and the time block processed",
         buckets: [0.5, 1, 2, 4, 6, 12],
       }),
+      receivedToGossipValidate: register.histogram({
+        name: "lodestar_gossip_block_received_to_gossip_validate",
+        help: "Time elapsed between block received and block validated",
+        buckets: [0.2, 0.5, 1, 1.5, 2, 4],
```

Comment:
I would add more granularity in all buckets: `[0.05, 0.1, 0.2, 0.5, 1, 1.5, 2, 4]`